### PR TITLE
Adding pub and sub AudioListeners

### PR DESCRIPTION
**Pull request by marinaserranomontes**
## Pull request checklist

- [ ] All tests pass. Demo project builds and runs.
- [ ] I have resolved any merge conflicts. Confirmation: ____

## What's in this pull request?

>...

### CODE REVIEW

File: accelerator-core/src/main/java/com/tokbox/android/otsdkwrapper/wrapper/OTWrapper.java [hunk 1]
```diff
@@ -488,28 +491,30 @@ public void removeRemote(String remoteId) {
...
-  public OTConfig getOTConfig(){
+  public OTConfig getOTConfig() {
     return this.mOTConfig;
   }
 
   private RetriableOTListener getUnfailingFromBaseListener(BaseOTListener listener) {
     return listener instanceof BasicListener ?
-      new UnfailingBasicListener((BasicListener) listener) :
-      new UnfailingAdvancedListener<>((AdvancedListener) listener);
+            new UnfailingBasicListener((BasicListener) listener) :
```
Comment (AntonioMA) [inline accelerator-core/src/main/java/com/tokbox/android/otsdkwrapper/wrapper/OTWrapper.java]: Indentation?

File: accelerator-core/src/main/java/com/tokbox/android/otsdkwrapper/wrapper/OTWrapper.java [hunk 2]
```diff
@@ -52,7 +52,8 @@
   private static final String LOG_TAG = OTWrapper.class.getSimpleName();
   private static final short LOCAL_LOG_LEVEL = 0xFF;
   private static final LogWrapper LOG =
-    new LogWrapper((short)(GlobalLogLevel.sMaxLogLevel & LOCAL_LOG_LEVEL));
+          new LogWrapper((short) (GlobalLogLevel.sMaxLogLevel & LOCAL_LOG_LEVEL));
```
Comment (AntonioMA) [inline accelerator-core/src/main/java/com/tokbox/android/otsdkwrapper/wrapper/OTWrapper.java]: Indentation?

File: accelerator-core/src/main/java/com/tokbox/android/otsdkwrapper/wrapper/OTWrapper.java [hunk 3]
```diff
@@ -619,38 +630,40 @@ public StreamStatus getLocalStreamStatus() {
         videoWidth = stream.getVideoWidth();
       }
       return new StreamStatus(mPublisher.getView(),
-                              mPublisher.getPublishAudio(), mPublisher.getPublishVideo(),
-                              hasAudio, hasVideo, streamVideoType,
-                              videoWidth, videoHeight);
+              mPublisher.getPublishAudio(), mPublisher.getPublishVideo(),
```
Comment (AntonioMA) [inline accelerator-core/src/main/java/com/tokbox/android/otsdkwrapper/wrapper/OTWrapper.java]: Indent? (usually this is indented to the level of the parameters)

File: accelerator-core/src/main/java/com/tokbox/android/otsdkwrapper/wrapper/OTWrapper.java [hunk 4]
```diff
@@ -619,38 +630,40 @@ public StreamStatus getLocalStreamStatus() {
...
+   * video
    */
   public StreamStatus getRemoteStreamStatus(String id) {
     Subscriber sub = mSubscribers.get(id);
     if (sub != null) {
       Stream subSt = sub.getStream();
       return new StreamStatus(sub.getView(), sub.getSubscribeToAudio(), sub.getSubscribeToVideo(),
-                              subSt.hasAudio(), subSt.hasVideo(), subSt.getStreamVideoType(),
-                              subSt.getVideoWidth(), subSt.getVideoHeight());
+              subSt.hasAudio(), subSt.hasVideo(), subSt.getStreamVideoType(),
```
Comment (AntonioMA) [inline accelerator-core/src/main/java/com/tokbox/android/otsdkwrapper/wrapper/OTWrapper.java]: Indentation?

File: accelerator-core/src/main/java/com/tokbox/android/otsdkwrapper/wrapper/OTWrapper.java [hunk 5]
```diff
@@ -813,7 +831,7 @@ private void removeOTListener(BaseOTListener listener, HashMap retriableMap,
 
   private synchronized void publishIfReady() {
     LOG.d(LOG_TAG, "publishIfReady: ", mSessionConnection, ", ", mPublisher, ", ",
-      startPublishing, ", ", isPreviewing);
+            startPublishing, ", ", isPreviewing);
```
Comment (AntonioMA) [inline accelerator-core/src/main/java/com/tokbox/android/otsdkwrapper/wrapper/OTWrapper.java]: Indentation? (one space less)

File: accelerator-core/src/main/java/com/tokbox/android/otsdkwrapper/wrapper/OTWrapper.java [hunk 6]
```diff
@@ -827,10 +845,10 @@ private synchronized void publishIfReady() {
     }
   }
 
-  private synchronized  void publishIfScreenReady(){
+  private synchronized void publishIfScreenReady() {
     LOG.d(LOG_TAG, "publishIfScreenReady: ", mSessionConnection, ", ", mScreenPublisher, ", ",
-      startSharingScreen);
-    if (mSession!= null && mSessionConnection != null && mScreenPublisher != null && startSharingScreen) {
+            startSharingScreen);
```
Comment (AntonioMA) [inline accelerator-core/src/main/java/com/tokbox/android/otsdkwrapper/wrapper/OTWrapper.java]: Indentation is strange here also

**Comment by marinaserranomontes** [comment]
It needs a double check

**Pull request closed without merging**